screen/settings/Shortcuts.tsx

```
@@ -0,0 +1,157 @@
+  const { wallets } = useStorage();
+  const { navigate } = useExtendedNavigation();
+  const { name } = useRoute();
+  const { setSelectedDefaultWallet, getSelectedDefaultWallet } = useOnAppLaunch();
+  const { receiveBitcoinIntent, setReceiveBitcoinIntent } = useSettings();
+  const [isSupportedEnvironment, setIsSupportedEnvironment] = useState(false);
+  const [defaultWallet, setDefaultWallet] = useState<TWallet | null>(null);
+
+  useEffect(() => {
+    const checkEnvironment = async () => {
+      const systemVersion = await DeviceInfo.getSystemVersion();
+      const systemVersion = await DeviceInfo.getSystemVersion();
```

Biome

[error] Shouldn't redeclare 'systemVersion'. Consider to delete it or rename it.

'systemVersion' is defined here:


(lint/suspicious/noRedeclare)